### PR TITLE
Ta 156 teacher can add new exercises 5

### PR DESCRIPTION
**Pull request by Hoolio333**
Re: [https://github.com/boolean-uk/team-dev-server/issues/156](url)

### CODE REVIEW

File: prisma/migrations/20220708143710_updated_schema/migration.sql [hunk 1]
```diff
@@ -1,27 +0,0 @@
--- CreateTable
```
Comment (dearshrewdwit) [inline prisma/migrations/20220708143710_updated_schema/migration.sql:1]: This is the migration that has already happened. Migrations that have been run should not be deleted, please undo this change.

File: src/domain/exercise.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+  const createdExercise = await dbClient.exercise.create({
+    data: { exerciseName, githubUrl }
+  })
+
+  return new Exercise(createdExercise.id, exerciseName, githubUrl)
+}
+
+export async function getExercises() {
+  const Exercises = await dbClient.exercise.findMany({
+    // include: { cohortExercises: true }
```
[the remark this answers is not in the document]
Comment (Hoolio333) [reply src/domain/exercise.js:21]: Removed.

File: src/domain/exercise.js [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+  const createdExercise = await dbClient.exercise.create({
+    data: { exerciseName, githubUrl }
+  })
+
+  return new Exercise(createdExercise.id, exerciseName, githubUrl)
+}
+
+export async function getExercises() {
+  const Exercises = await dbClient.exercise.findMany({
```
[the remark this answers is not in the document]
Comment (Hoolio333) [reply src/domain/exercise.js:20]: Changed, thanks.

**Review by Dan2024 — APPROVED** [review]
